diff -ur ivman-0.6.14.orig/src/manager.c ivman-0.6.14/src/manager.c
--- ivman-0.6.14.orig/src/manager.c	2007-01-29 09:52:31.000000000 -0200
+++ ivman-0.6.14/src/manager.c	2008-01-21 15:55:10.000000000 -0200
@@ -280,7 +280,7 @@
                        "used for mounting.\n"));
                 known_mount_command = "pmount-hal '$hal.info.udi$'";
                 unknown_mount_command = known_mount_command;
-                umount_command = "pumount '$hal.block.device$'";
+                umount_command = "pumount -l '$hal.block.device$'";
                 return;
             }
         }
@@ -326,7 +326,7 @@
                     "pmount '$hal.block.device$' "
                     "'$hal.volume.policy.desired_mount_point$'";
                 unknown_mount_command = "pmount '$hal.block.device$'";
-                umount_command = "pumount '$hal.block.device$'";
+                umount_command = "pumount -l '$hal.block.device$'";
                 return;
             }
 
@@ -361,7 +361,7 @@
                 unknown_mount_command = command->str;
                 g_string_free(command, FALSE);
 
-                umount_command = "pumount '$hal.block.device$'";
+                umount_command = "pumount -l '$hal.block.device$'";
                 return;
             }
 
